src/bundle/Controller/SearchController.php

```diff
@@ -55,7 +56,7 @@ class SearchController extends Controller
 
     /**
      * @param \eZ\Publish\API\Repository\SearchService $searchService
-     * @param \EzSystems\EzPlatformAdminUi\Tab\Dashboard\PagerContentToDataMapper $pagerContentToDataMapper
+     * @param \EzSystems\EzPlatformAdminUi\Search\PagerContentToDataMapper $pagerSearchContentToDataMapper
```

Comment:
Wrong type, should be `\EzSystems\EzPlatformAdminUi\Search\PagerSearchContentToDataMapper`